src/components/EventCalendar/EventsInMonth.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { createUseStyles } from 'react-jss'
+import { useTranslation } from 'react-i18next'
+import CalendarEventPanel from './CalendarEventPanel'
+import { CalendarEventDataFragment } from '../../graphql/__generated__/typescript-operations'
```

Comment:
Tady by u importu bylo lehce lepší používat absolutní cestu (předpokládám, že to takhle dalo IDE)